src/main/java/org/jabref/logic/exporter/MSBibExporter.java

```diff
@@ -19,33 +19,37 @@
 import org.jabref.model.database.BibDatabaseContext;
 import org.jabref.model.entry.BibEntry;
 
+import org.jspecify.annotations.NonNull;
+
 /**
  * TemplateExporter for exporting in MSBIB XML format.
  */
 class MSBibExporter extends Exporter {
 
+    private final TransformerFactory transformerFactory;
+
     public MSBibExporter() {
         super("MSBib", "MS Office 2007", StandardFileType.XML);
+        transformerFactory = TransformerFactory.newInstance();
     }
 
     @Override
-    public void export(final BibDatabaseContext databaseContext, final Path file,
-                       List<BibEntry> entries) throws SaveException {
-        Objects.requireNonNull(databaseContext);
-        Objects.requireNonNull(entries);
-
+    public void export(@NonNull BibDatabaseContext databaseContext,
+                       @NonNull Path file,
+                       @NonNull List<BibEntry> entries) throws SaveException {
+        Objects.requireNonNull(databaseContext); // required by test case
```

Comment:
This is somekind of silly, isn't it?

Reply:
Future work would be to introduce JSPecify more globally - but I did not want to escalate the PR here.